frontend/src/components/stages/election_view.ts

```diff
@@ -94,18 +121,25 @@ export class ElectionView extends MobxLitElement {
     `;
   }
 
-  private renderDraggableParticipant(profile: ParticipantProfile) {
-    if (profile.publicId === this.participantService.profile?.publicId) {
-      return nothing;
-    }
+  private renderElectionItem(item: ElectionItem) {
+    // TODO: Allow item photos?
+    return html`
+        <div class="right">
```

Comment:
Missing <div class="participant"> between line 126, 127

Reply:
Done, thank you!